keylime/web_util.py

```diff
@@ -190,6 +190,16 @@ def get_restful_params(urlstring):
     return path_params
 
 
+def validate_api_version(handler, version, logger):
+    if not version or not keylime_api_version.is_supported_version(version):
+        echo_json_response(handler, 400, "API Version not supported")
+        return False
+
+    if keylime_api_version.is_deprecated_version(version):
+        logger.warning('Client request to API version %s is deprecated and will be removed in future versions.', version)
```

Comment:
We should sanitize the API version before logging, but otherwise the PR looks fine to me.

Reply:
That's already happening in `get_restful_params()` as part of pulling it from the URL. It's sanitized and normalized.

Reply:
Yes, you are right. I missed that.